systemd/osrt-docker-publisher.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=openSUSE Release Tools: Docker image publisher
+
+[Timer]
+OnBootSec=120
+OnUnitInactiveSec=15 min
```

Comment:
Wouldn't it be about time to move to event-based triggering based on OBS/openQA events? See https://rabbit.opensuse.org/ and https://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.administration.html#_message_bus for details

Reply:
That alone won't be sufficient - if somehow the TW release message gets lost (slow network, packet loss, downtime...), the bot would never release the containers.

The script itself can already be called to only release one image, listening for events should be done externally then.

Reply:
If you rebase you can drop the packaging and the systemd units and replace it with a gocd trigger on the Factory release pipeline.